client/src/components/TopSongGallary/TopSongGallary.jsx

```diff
@@ -6,52 +6,68 @@ import {
   TopHSongCountainer,
 } from "./TopSongGallaryStyle";
 import cover1 from "../../assets/TopSongGallary/Rectangle 3.png";
-import cover2 from "../../assets/TopSongGallary/Rectangle 2.png";
 import cover4 from "../../assets/TopSongGallary/Rectangle 1.png";
-import cover5 from "../../assets/TopSongGallary/Rectangle 4.png";
 import ImageBoxWithDetails from './ImageBoxWithDetails'
+import { useEffect, useState } from "react";
+
 export default function TopSongGallary() {
+  
+  const [ topArabicSingers, setTopArabicSingers ] = useState([])
+  const [ topArabicSongs, setTopArabicSongs ] = useState([])
+  const [ topHebrewSongs, setTopHebrewSongs ] = useState([])
+
+  async function fetchTopArabicSongs() {
+    try {
+      const response = await fetch('http://localhost:5000/topArabicSongs');
+      const data = await response.json();
+      setTopArabicSingers(Object.keys(data))
+      setTopArabicSongs(Object.values(data))
+    } catch (error) {
+      console.error('Error:', error);
+    }
+  }
+  
+  async function fetchTopHebrewSongs() {
```

Comment:
this should be with rtk query, and should not be defined here. plus, save url as constant